conandata.yml

```diff
@@ -50,10 +51,30 @@ pyinstaller:
           package: "native_cad_plugin"
           src: "res/bundled_packages"
           dst: "share/cura/resources/bundled_packages"
-        cura_resources:
-            package: "cura"
-            src: "resources"
-            dst: "share/cura/resources"
+        cura_resources_definitions:
+            package: "cura_resources"
+            src: "res/definitions"
+            dst: "share/cura/resources/definitions"
+        cura_resources_extruders:
+            package: "cura_resources"
+            src: "res/extruders"
+            dst: "share/cura/resources/extruders"
+        cura_resources_intent:
+            package: "cura_resources"
+            src: "res/intent"
+            dst: "share/cura/resources/intent"
+        cura_resources_meshes:
+            package: "cura_resources"
+            src: "res/meshes"
+            dst: "share/cura/resources/meshes"
+        cura_resources_quality:
+          package: "cura_resources"
+          src: "res/quality"
+          dst: "share/cura/resources/quality"
+        cura_resources_variants:
+          package: "cura_resources"
+          src: "res/variants"
+          dst: "share/cura/resources/variants"
```

Comment:
No way to define those all in a single shot ?

Reply:
Should be possible